pkg/operator/configobserver/etcd/observe_etcd_endpoints.go

```diff
@@ -106,6 +107,7 @@ func innerObserveStorageURLs(fallbackObserver fallBackObserverFn, alwaysAppendLo
 		}
 		// skip placeholder ip addresses used in previous versions where the hostname was used instead
 		if strings.HasPrefix(ip.String(), "192.0.2.") || strings.HasPrefix(ip.String(), "2001:db8:") {
```

Comment:
can these addresses be provided manually by a user? 

See https://coreos.slack.com/archives/CB48XQ4KZ/p1606731694493500?thread_ts=1606390917.467300&cid=CB48XQ4KZ for context

Reply:
> // skip placeholder ip addresses used in previous versions where the hostname was used instead

OK I understand, interesting. Since we dropped the pattern in 4.6+ of populating host-etcd with predefined endpoints including the placeholder addresses I think we should just remove the check and tolerate the address? If OpenShift were to enforce RFC 5735, 5737 it should be done by the installer or network-operator IMO. What do you think?

Reply:
Ok, it makes sense. I will update the PR accordingly.

Reply:
done, PTAL.